### PR TITLE
More sync options

### PR DESCRIPTION
**Pull request by neube3**
Implemented most of #150 ("excluding files from sync") requests and including-override added. Note that it still lacks rsync-style (include|exclude)-from=FILE

### CODE REVIEW

**Comment by bwbeach** [comment]
Nice work.

If I'm understanding correctly, `--includeRegex` only makes sense when you have specified `--excludeRegex`.  Would it be a good idea to sanity check the args, and report an error if `--includeRegex` is used alone?

**Comment by neube3** [comment]
Applied the requested changes.

File: b2/console_tool.py [hunk 1]
```diff
@@ -590,7 +590,7 @@ class Sync(Command):
     """
     b2 sync [--delete] [--keepDays N] [--skipNewer] [--replaceNewer] \\
             [--compareVersions <option>] [--threads N] [--noProgress] \\
-            [--excludeRegex <regex>] <source> <destination>
+            [--excludeRegex <regex>] [--includeRegex <regex>] <source> <destination>
```
Comment (ppolewicz) [inline b2/console_tool.py]: `[--excludeRegex <regex> [--includeRegex <regex>]]`

Comment (neube3) [inline b2/console_tool.py]: Applied requested changes; added note to Sync class documentation about --includeRegex requiring --excludeRegex for clarity.

**Pull request merged**